index.js

```diff
@@ -119,6 +119,47 @@ Vec3.prototype.min = function(other) {
 Vec3.prototype.max = function(other) {
   return new Vec3(Math.max(this.x, other.x), Math.max(this.y, other.y), Math.max(this.z, other.z));
 };
+Vec3.prototype.norm = function() {
+  return Math.sqrt(this.x * this.x + this.y * this.y + this.z * this.z);
+};
+Vec3.prototype.unit = function() {
+  var norm = this.norm();
+  if(norm === 0) {
+    return this;
+  } else {
```

Comment:
Would be better to return a new vector here to he coherent with the other case (callers should be able to assume mutability or immutability)